### ISSUE TITLE
Importlib broken with Python3.8 and cx_Freeze 6.10

### ISSUE THREAD

**Issue by robbertjanpijpker** [marcelotduarte/cx_Freeze#1362]
Since the upgrade to cx_Freeze 6.10, our build system stopped producing a working executable, and produces now a ModuleNotFoundError. Reverting to cx_Freeze 6.9 works, however, we would like to be able to upgrade to newer (and probably better versions of cx_Freeze).

I've reproduced this error with a minimal example, I hope that that helps. This minimal example works with cx_Freeze 6.9, but breaks with cx_Freeze 6.10. I'm working on Windows 10, using Python 3.8.3 and also had this issue with Python 3.9.7.

## main.py
```Python
import importlib

if __name__ == "__main__":
    module = importlib.import_module("module")
    module.hello_world()
```

## module.py
```Python
def hello_world():
    print("Hello World")
```

## setup.py
```Python
from cx_Freeze import setup, Executable

build_exe_options = {"include_files": ["module.py"]}

base = None

setup(
    name = "app",
    version = "0.1",
    options = {"build_exe": build_exe_options},
    executables = [Executable("main.py", base=base)]
)
```

**Comment by marcelotduarte** [comment]
If you change "[include_files](https://cx-freeze.readthedocs.io/en/latest/setup_script.html#cmdoption-arg-include_files)" to "[includes](https://cx-freeze.readthedocs.io/en/latest/setup_script.html#cmdoption-arg-includes)" it will work. There's been no change to that recently, it's a pretty old option.
Knowing it was going to go wrong, out of curiosity I tested your code in cx_Freeze 6.9 with Python 3.9.7 on my Linux machine.
Also note that if your actual code is a package, you have to use [packages](https://cx-freeze.readthedocs.io/en/latest/setup_script.html#cmdoption-arg-packages).
See also [this](https://cx-freeze.readthedocs.io/en/latest/faq.html#specifying-modules-and-packages).

EDIT:
You should change:
`build_exe_options = {"include_files": ["module.py"]}`
to
`build_exe_options = {"includes": ["module"]}`

I tested it in Windows and confirm your findings, but I can't revert the behavior because it is a security issue. In Linux I already implemented in cx_Freeze 6.2 or 6.3... and only now I patched the windows version.
For more info see PR #1324 and the linking to python docs that links to CVE.

**Comment by robbertjanpijpker** [comment]
Thanks for your quick answer. I didnt know this was changed because it was originally a security issue, and thus missed it in 6.10's release notes. I'll update it to use `includes` from now on. Keep up the good work! 😄
